modules/pam_usertype/Makefile.am

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
```

Comment:
As this is derived from other makefiles, you should have kept the original copyright notice and just add the new one.

Reply:
Done.